### PR TITLE
fix: Removed back button from Role page

### LINKED ISSUE #20818: [Design] The role page has a back button but it shouldn’t
### Steps to reproduce
- Go to U&P plugin > Roles
- See the Back button

### Expected behavior

There should not be a back button on this page

### Screenshots

CleanShot 2024-07-04 at 17 54 04@2x

### PR DESCRIPTION
**Pull request by Sam-Phillemon9493**
What does it do?
Removed the back button from the Roles page.

Why is it needed?
To fix issue [20818](https://github.com/strapi/strapi/issues/20818): The role page has a back button but it shouldn’t 

How to test it?

- Go to U&P plugin > Roles
- See the Back button

Related issue(s)/PR(s)
fixes #20818

### CODE REVIEW

**Comment by hanpaine** [comment]
hey @Sam-Phillemon9493 thanks for the contribution - we'll take a look v soon 🚀

**Comment by hanpaine** [comment]
(updated the description to add "fixes" so it will auto-close the Issue 😄 )

File: packages/plugins/users-permissions/admin/src/pages/Roles/pages/ListPage/index.jsx [hunk 1]
```diff
@@ -186,8 +185,7 @@ export const RolesListPage = () => {
                 })}
               </LinkButton>
             ) : null
-          }
-          navigationAction={<BackButton />}
+          }
```
Comment (butcherZ) [inline packages/plugins/users-permissions/admin/src/pages/Roles/pages/ListPage/index.jsx:188]: Hey Sam, thank you for your contribution! I performed QA, and it's working.

Just one thing, could you run `yarn prettier:write` at the project root and `yarn lint` again before submitting your code? It seems like line 188 has some whitespaces, and it won't pass our linter. Once this is done we can merge it directly into develop. thank you! 😄

**Review by butcherZ — CHANGES_REQUESTED** [review]

File: packages/plugins/users-permissions/admin/src/pages/Roles/pages/ListPage/index.jsx (hunk 1 above)
Comment (Sam-Phillemon9493) [reply packages/plugins/users-permissions/admin/src/pages/Roles/pages/ListPage/index.jsx]: @butcherZ , Thank you!  I’ve made the requested updates. Please let me know if there’s anything else is needed before merging!

**Review by butcherZ — APPROVED** [review]
LGTM :) Thanks again @Sam-Phillemon9493!

**Pull request merged**